CHANGELOG.md

```diff
@@ -75,13 +76,14 @@ No changes since `@mui/x-tree-view@v7.22.3`.
 
 ### Docs
 
-- [docs] Add data caching to lazy loaded detail panel demo (@cherniavskii) (#15555)
-- [docs] Remove selectors section from list view docs (@KenanYusuf) (#15639)
+- [docs] Add data caching to lazy loaded detail panel demo (#15555) @cherniavskii
+- [docs] Remove selectors section from list view docs (#15639) @KenanYusuf
+- [docs][data grid] Add documentation for the list view feature (#15344) (#15635) @KenanYusuf
 
 ### Core
 
 - [core] Update @mui/monorepo (#15574) @oliviertassinari
-- React 19 support (@arminmeh) (#15557)
+- React 19 support (#15557) @arminmeh
```

Comment:
Last week in the alpha release we added this one to every set of components and removed from `Core`
You can see the diff [here](https://github.com/mui/mui-x/pull/15554/files)

Reply:
I have added the suggestions for all packages